runway/data_types.py

```diff
@@ -476,3 +476,126 @@ def to_dict(self):
         if self.is_directory: ret['isDirectory'] = self.is_directory
         if self.extension: ret['extension'] = self.extension
         return ret
+
+
+class segmentation(BaseType):
+    """A datatype that represents a pixel-level segmentation of an image.
+    Each pixel is annotated with a label id from 0-255, each corresponding to a
+    different object class.
+    When used as an input data type, `segmentation` accepts either a two-dimensional array,
+    a 1-channel base64-encoded PNG image, or a 3-channel base64-encoded PNG colormap image.
```

Comment:
What do the three channels correspond to if a 3-channel base64-encoded PNG image is used? This is somewhat unclear upon first reading.

Reply:
Agreed it's unclear. If a 3-channel image is passed, it assumes that it's a colormap image, so it uses the colors from the `label_to_color` dictionary. I'll try to clarify.